app/ui-utils/client/lib/RoomHistoryManager.js

```diff
@@ -233,8 +224,9 @@ export const RoomHistoryManager = new (class extends Emitter {
 		}
 
 		waitAfterFlush(() => {
-			const heightDiff = wrapper.scrollHeight - previousHeight;
-			wrapper.scrollTop = scroll + heightDiff;
+			lastMessage &&
+				($(`#${lastMessage._id}`).addClass('highlight'),
+				document.getElementById(lastMessage._id).scrollIntoView({ block: 'end', inline: 'nearest' }));
```

Comment:
Hm can you simplify this a little bit? Best to do `if (!lastMessage) return;` then the rest. Also, can we not store `$('#' + lastMessage._id, wrapper)` and get the `Element` from there instead if calling `document.getElementById` again?